server.py

```diff
@@ -27,7 +28,16 @@ def extractingIP():
 
 # ----------------------- Constants and Global Variables -----------------------
 
-DEBUG=False
+#logs and debugs
+DEBUG_M_IN=False
```

Comment:
It would be nice to have a little comment on what each debug variable manage.

Otherwise, the code looks good to me !